smac/utils/io/cmd_reader.py

```diff
@@ -10,8 +10,72 @@
 __version__ = "0.0.1"
 
 
-class CMDReader(object):
+help_type = "standard"
+
+
+class ConfigurableHelpFormatter(ArgumentDefaultsHelpFormatter):
+    """
+    Configurable Help Formatter. Can filter out developer options.
+    """
+
+    def __init(self, *args, **kwargs):
+        super(ConfigurableHelpFormatter, self).__init__(*args, **kwargs)
+
+    def _add_item(self, func, args):
+        def filter_actions(actions):
+            filtered_actions = []
+            for action in actions:
+                if isinstance(action, Action):
```

Comment:
is this type check here necessary? 
(in general, please try to avoid type checks if possible)

Reply:
It is not necessary, I removed it.